libvips/foreign/nsgifload.c

```diff
@@ -332,7 +332,7 @@ vips_foreign_load_nsgif_set_header( VipsForeignLoadNsgif *gif,
 		}
 	}
 
-	vips_image_set_int( image, "palette-bit-depth", 
+	vips_image_set_int( image, VIPS_META_BITS_PER_SAMPLE, 
```

Comment:
I think we still need to set the `palette-bit-depth` field for compatibility.

Reply:
I fear I miunderstood how deprecation works. I added the deleted code lines again.